<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7" stretch="100,0,0">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>PId do Processo</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLineEdit" name="lineEdit_PID"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnKill">
               <property name="text">
                <string>Kill</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnStop">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bntCount">
               <property name="text">
                <string>Cont</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="30,70,0">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Prioridade</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdi_Prioridade"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnPrioridade">
               <property name="text">
                <string>Aplicar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="30,70,0">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>CPU</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_CPU"/>
             </item>
             <item>
              <widget class="QPushButton" name="btncpu">
               <property name="text">
                <string>Aplicar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Filtro</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_filtro"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>CARGA de CPU</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="20,80,0">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>CPU 0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar_0">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>10</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>89</number>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_13">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="20,80,0">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>CPU 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar_1">
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_14">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="20,80,0">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>CPU 2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar_2">
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_15">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,20,80">
               <item>
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>CPU 3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar_3">
                 <property name="value">
                  <number>24</number>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,20,80">
               <item>
                <spacer name="horizontalSpacer_11">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>CPU 4</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar_4">
                 <property name="value">
                  <number>24</number>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="0,20,80">
               <item>
                <spacer name="horizontalSpacer_12">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>CPU 5</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="progressBar_5">
                 <property name="value">
                  <number>24</number>
                 </property>
                 <property name="invertedAppearance">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>SAIR</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectPID(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_filtro</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>processFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bntCount</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnKill</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>kill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrioridade</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>priorityProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btncpu</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuAllocate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copiaTexto()</slot>
  <slot>process()</slot>
  <slot>start()</slot>
  <slot>quit()</slot>
  <slot>processFilter()</slot>
  <slot>usingCPU()</slot>
  <slot>kill()</slot>
  <slot>stop()</slot>
  <slot>cont()</slot>
  <slot>selectPID(int,int)</slot>
  <slot>priorityProcess()</slot>
  <slot>cpuAllocate()</slot>
  <slot>getStatusPID()</slot>
 </slots>
</ui>
